src/transaction/log_append.cpp

```diff
@@ -1735,7 +1735,7 @@ prior_list_deserialize (const std::string &str, log_prior_node *&head, log_prior
 static void
 prior_lsa_start_append (THREAD_ENTRY *thread_p, LOG_PRIOR_NODE *node, LOG_TDES *tdes)
 {
-  assertm (get_server_type () == SERVER_TYPE_TRANSACTION, "Log append can be executed only on transaction server");
+  assertm (get_server_type () != SERVER_TYPE_PAGE, "Log append can be executed only on transaction server");
```

Comment:
I do not understand the context in which this function is called, but at least the text should be updated.

Reply:
I changed the logic so it would work on servers that start as UNKNOWN, the text can be changed to `Log append can not be executed only on page server`